### PR TITLE
Update : [hac-dev] added a periodic job  

### PR DESCRIPTION
**Pull request by sawood14012**
changes 
- Add the QE members as reviewers 
- Add a Periodic Job to test 
- Add Specific Plugins needed

### CODE REVIEW

File: ci-operator/jobs/openshift/hac-dev/OWNERS [hunk 1]
```diff
@@ -18,3 +18,4 @@ reviewers:
 - rottencandy
 - sahil143
 - vikram-raj
+- rhopp
```
Comment (flacatus) [inline ci-operator/jobs/openshift/hac-dev/OWNERS:21]: New line

Comment (sawood14012) [reply ci-operator/jobs/openshift/hac-dev/OWNERS]: done

File: ci-operator/config/openshift/hac-dev/OWNERS [hunk 2]
```diff
@@ -18,3 +18,6 @@ reviewers:
 - rottencandy
 - sahil143
 - vikram-raj
+- rhopp
```
Comment (rohitkrai03) [inline ci-operator/config/openshift/hac-dev/OWNERS:21]: You shouldn't edit this file. This is auto generated from the source repository.

File: ci-operator/jobs/openshift/hac-dev/OWNERS [hunk 3]
```diff
@@ -18,3 +18,5 @@ reviewers:
...
```
Comment (rohitkrai03) [inline ci-operator/jobs/openshift/hac-dev/OWNERS:21]: Same as above.

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/hac-dev/OWNERS (hunk 2 above)
Comment (sawood14012) [reply ci-operator/config/openshift/hac-dev/OWNERS:21]: ohh my bad let me fix it

**Review by rohitkrai03 — APPROVED** [review]
/lgtm
/approve

**Pull request merged**